src/compas_fab/robots/planning_scene.py

```diff
@@ -119,7 +119,7 @@ def __data__(self):
 
 
 class AttachedCollisionMesh(Data):
-    """Represents a collision mesh that is attached to a :class:`Robot`'s :class:`~compas_robots.model.Link`.
+    """Represents a collision mesh that is attached to a :class:`Robot`'s :class:`compas_robots.model.Link`.
```

Comment:
I actually like this with the `~` better, otherwise it is too verbose, don't you think?

Reply:
Yea. But is it clear to people that compas_robots is a different package? At the moment if you pretend you are a user following that clickable link, you will reach a page that simply list the class Link. Somehow misleading that the Link class belongs to compas_fab.

I think if the API documentation page theme is slightly different between compas_fab and compas_robots, (maybe a different colour) then we can go with ~ for all the compas_robot links.

Reply:
You're right, it's useful to keep the full link in these cases, to highlight it's an external module. Altering the colors of fab would be a bit against the idea of having homogenous docs for all compas, so, let's stick with full link